docs/examples/paddle/paddle-basic_example.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
We have `qa/TL1_jupyter_plugins/test.sh` that test jupyters. Please add PaddlePaddle to `pip_packages` so test will install it before running jupyter.
Please add the exact version of PaddlePaddle you want to test against in https://github.com/NVIDIA/DALI/blob/master/qa/setup_packages.py#L23.

Reply:
Done. Note that the paddle patch needed was still in `develop` tree, so I included the dev whl packages, will update to 1.6 when it is released in the next couple weeks. Also, paddle package does not use the cuXX naming convention, so I included full path.

Reply:
Sure. The way you did it is fine. We need to handle different FW, and every one has different naming convention. So as long as it works it is fine.